app/controller/partycontroller.ctrl.php

```diff
@@ -201,10 +201,29 @@ public function create()
                         }
 
                         if(hasRole($this->user, 'Host')){
+                            // send email to Admin
+                              $message = "<p>Hi,</p>" .
+                                       "<p>This is an automatic email to let you know that ". $name . " has created a party on the <strong>" . APPNAME . "</strong>.</p>" .
+                                        "<p><strong>Group Name:</strong> ". $group ." <p>" .
```

Comment:
$group is the group id (we should really rename this variable to $group_id...)
For group name, you will need to do something like:

    $group_obj = $Groups->findOne($group);
    $group_name = $group_obj->name;